docs/src/man/troubleshooting.md

```diff
@@ -83,14 +83,14 @@ define i64 @julia_foo_62405(%CuDeviceArray.2* nocapture readonly, i64) {
 ```
 
 
-## Debug info and line-number information
+## Debug @info and line-number information
```

Comment:
nonono

Reply:
yeah, I was meaning to run over the diff ;) It's easier on github!